### PR TITLE
issue 611: rename Larray class as Array

### PR DESCRIPTION
**Pull request by alixdamman**
Waiting for PR #809 to be merged before to finish this PR.

### CODE REVIEW

**Comment by alixdamman** [comment]
The first and the three last commits are the most important.

File: larray/core/array.py [hunk 1]
```diff
@@ -8027,63 +8027,69 @@ def transform(v):
         return self.apply(transform, dtype=dtype)
 
 
+class LArray(Array):
+    def __init__(self, *args, **kwargs):
+        warnings.warn("LArray has been renamed as Array.", FutureWarning, stacklevel=2)
+        Array.__init__(self, *args, **kwargs)
```
Comment (gdementen) [inline larray/core/array.py]: can't we use `LArray = renamed_to(Array, 'LArray')` instead?

Comment (alixdamman) [reply larray/core/array.py]: ``rename_to`` return a function and not a class. In addition, the printed warning message would have been `LArray() is deprecated. Use Array() instead`, i.e. with ( ) after class names.

Comment (gdementen) [reply larray/core/array.py]: Calling a function and a class is essentially the same thing in Python (they are both callables) and I don't see the problem with that warning message.

**Pull request merged**